### PR TITLE
Complete mobile version

### PR DESCRIPTION
**Pull request by alizaamir123**
In this PR:
Create Index.html
Create Style.css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+  <li class="twitter"><a href=""><img src="./images/gmail.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/github.png" alt=""></a></li>
+  <li class="twitter"><a href=""><img src="./images/Vector.png" alt=""></a></li>
+
+</ul>
+<button class="get-resume">get my resume</button>
+  </div>
+<div >
+  <div class="language-section" >
+    <p class="languages">Languages </p>
```
Comment (Nyame-Wolf) [inline index.html:145]: Kindly ensure that you use the semantic tag `<ul>`. This should be a list of lists, that is language, Skills, and Frameworks are all items of a list
inside language `<li>` and have another list with javascript, HTML, and CSS.
Screenshot 2023-04-29 at 08 49 20

**Review by Nyame-Wolf — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle: 

Hi @alizaamir123 :wave: 

Your project page is looking fantastic so far :star_struck: ! 

However, there is still some room for improvement, that can help you have better code and make your project professional :sunglasses: 



## Highlights :sparkles:

- Great work using the right GitHub Flow :100: 
- Great work using the Figma design ☑️ 
- Commits are descriptive and meaningful :+1: 
- Project Documentation (PR, README) is professional :necktie: 

Congrats on your work so far! :clap:  

## Required Changes ♻️

There are some issues that you still need to work on to go to the next project, but you are almost there! 
_For more information, check the comments under the review :arrow_down:_

## Optional suggestions :ballot_box_with_check: 

_Every comment with the **[OPTIONAL]** prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can improve the quality of your code :100:_

Happy coding and keep it up! :raised_hands: 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
  Please, remember to tag me **@Nyame-Wolf**  in your question so I can receive the notification.

  **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------

  _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            <ul class="technologies">
+                <li class="html technologies1">Html</li>
+                <li class="Css technologies1">Css</li>
+                <li class="javascript technologies1">Javascript</li>
+            </ul>
+            <button class="see-project">See project</button>
+        </div>
+    </div>
+  </section>
+<section class="about-container">
```
Comment (ABDULALI3468) [inline index.html:131]: Please consider making about-container `display: flex` in the styles as asked by the previous code reviewer to meet the project requirements. 

## MORE ELABORATION:-

You just need to make them display flex and that's it:-

```
.about-container{
  display: flex;
  flex-direction: column;
}
```

File: index.html [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+      <li class="javascript-lang"><img src="./images/css.png" alt="">Css</li>
+      
+  </ul>
+</div>
+<ul class="after-about">
+  <li class="frameworks">Frameworks <img class="sidearrow1" src="./images/arrow.png" alt=""></li>
+  <img src="./images/sep.png" alt="">
+  <li class="frameworks">Skills <img class="sidearrow2" src="./images/arrow.png" alt=""></li>
+  <img src="./images/sep.png" alt="">
+</ul>
```
Comment (ABDULALI3468) [inline index.html:168]: As requested by the previous code reviewer kindly follow these instructions to meet the project requirements:-

Please consider following these 2 rules in this section as it's a good practice for semantics:-

1. The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li`
2. The technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` and each of them a `li`


## Solution with a detailed demonstration:-

Following is the code representation of the above if you are confused about what the instructions mean:-


```
    <ul>
      <li>
        Languages
        <ul>
          <li>JavaScript</li>
          <li>Html</li>
          <li>CSS</li>
        </ul>
      </li>
      <li>
        Frameworks
      </li>
      <li>
        Skills
      </li>
    </ul>
```

**Review by ABDULALI3468 — CHANGES_REQUESTED** [review]
# Changes Required 👍 

Hi @alizaamir123 , 



Great work on making the changes requested by a previous reviewer 👏 
You've done really well implementing some of the requested changes, but there are still some which aren't been addressed yet.


## Highlights
☑️Implemented changes asked by the previous CR.
✅All linters are checked.
✅Project is professionally documented.
✅The right GitHub flow method has been used.
✅Professional pull request title and description.


 ## Required Changes ♻️

> **_Check the comments below to have a look at the required changes._**

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 4]
```diff
@@ -1 +1,113 @@
...
+"Special thanks to "Muneeb Rehman" for helping me in this project. His dedication, skills, and collaborative spirit were essential to my success."
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](LICENSE.md) licensed.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (ademibowale) [inline README.md:113]: - [ ] [Optional]  Kindly add the correct path of the MIT LICENSE in your project, as I can see you have not added the path correctly the link not working . Please look at the below suggestion to solve the issues.
Correct mit license readme

**Review by ademibowale — CHANGES_REQUESTED** [review]
# Required Changes 🔁
# Hello @alizaamir123     😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

just a little bit

## Highlights
  ✅ Nice user interface
  ✅Correct use of Github Flow
  ✅ No linter errors
✅ Descriptive Pull Request
  

## Required Changes ♻️




_Check the comments under the review._




## Optional suggestions 
   
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by PinkMoon25 — APPROVED** [review]
Hi @alizaamir123 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Changes Implemented :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**